lib/instrumentation/transaction.js

```diff
@@ -92,6 +93,11 @@ Transaction.prototype.setUserContext = function (context) {
   this._user = Object.assign(this._user || {}, context)
 }
 
+Transaction.prototype.setFaas = function (context) {
```

Comment:
nit: Perhaps change this "context" argument to "faas" or "faasFields" or something.

"context" is a very overloaded word in the agent code:

- the "context" object on spans on transactions
- async "context" in "async-hooks.js"
- trace-"context" for the W3C trace-context headers/data

Reply:
✅ in 61ec4029b86c843b9b2831991525cdd5405ac800